compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -1468,49 +1472,51 @@ private Result seekMatchInExpressionRhs(STToken nextToken, int lookahead, int cu
         switch (parentCtx) {
             case ARG_LIST:
                 next = new ParserRuleContext[] { ParserRuleContext.BINARY_OPERATOR, ParserRuleContext.DOT,
-                        ParserRuleContext.OPEN_BRACKET, ParserRuleContext.COMMA, ParserRuleContext.ARG_LIST_START,
-                        ParserRuleContext.ARG_LIST_END };
+                        ParserRuleContext.ANNOT_CHAINING_TOKEN, ParserRuleContext.OPEN_BRACKET, ParserRuleContext.COMMA,
+                        ParserRuleContext.ARG_LIST_START, ParserRuleContext.ARG_LIST_END };
```

Comment:
Will open a new ctx for access expression types in my next PR as this is repeated within this method.